cyipopt/tests/unit/test_scipy_optional.py

```diff
@@ -63,10 +63,28 @@ def test_minimize_ipopt_nojac_constraints_if_scipy():
     x0 = [1.3, 0.7, 0.8, 1.9, 1.2]
     constr = {"fun": lambda x: rosen(x) - 1.0, "type": "ineq"}
     res = cyipopt.minimize_ipopt(rosen, x0, constraints=constr)
+    print(res)
     assert isinstance(res, dict)
     assert np.isclose(res.get("fun"), 1.0)
     assert res.get("status") == 0
     assert res.get("success") is True
     expected_res = np.array([1.001867, 0.99434067, 1.05070075, 1.17906312,
                              1.38103001])
     np.testing.assert_allclose(res.get("x"), expected_res)
+
+@pytest.mark.skipif("scipy" not in sys.modules,
+                    reason="Test only valid if Scipy available.")
+def test_minimize_ipopt_nojac_constraints_vectorvalued_if_scipy():
+    """ `minimize_ipopt` works without Jacobian and with constraints 
+    without specifying the jacobian of the vector valued constraint 
+    function"""
+    from scipy.optimize import rosen, rosen_der
+    x0 = np.array([0.5, 0.75])
+    bounds = [np.array([0, 1]), np.array([-0.5, 2.0])]
+    expected_res = 0.25 * np.ones_like(x0)
+    eq_cons = {'fun' : lambda x: x - expected_res, 'type': 'eq'}
+    res = cyipopt.minimize_ipopt(rosen, x0, jac=rosen_der, bounds=bounds, constraints=[eq_cons])
+    assert isinstance(res, dict)
+    assert res.get("status") == 0
+    assert res.get("success") is True
+    np.testing.assert_allclose(res.get("x"), expected_res)
```

Comment:
```suggestion
    np.testing.assert_allclose(res.get("x"), expected_res)

```
Blank line at end of file.